### PR TITLE
startxiwi

### PR DESCRIPTION
**Pull request by DennisLfromGA**
Add `host-bin/startxiwi` wrapper script and change `targets/xiwi` to add it to HOSTBIN and TIPS.

### CODE REVIEW

File: targets/xiwi [hunk 1]
```diff
@@ -403,9 +404,9 @@ Once in a crouton window, press fullscreen or the "switch window" key to switch
 back to Chromium OS.
 
 You can launch individual apps in crouton windows by using the "xiwi" command
-in the chroot shell. Wrap that command with enter-chroot to launch directly
-from the host shell. Use the enter-chroot parameter -b to run in the background.
-Example: sudo enter-chroot -b xiwi xterm
+in the chroot shell. Use startxiwi to launch directly from the host shell.
+Use the startxiwi parameter -b to run in the background.
+Example: sudo startxiwi -b xiwi xterm
```
Comment (dnschneid) [inline targets/xiwi]: `xiwi` shouldn't be in that command

File: host-bin/startxiwi [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      c|k|n|t|u)
+         OPTARG="$(echo -n "$OPTARG" | sed -e "s/'/'\\\\\\''/g")"
+         OPTS_ENTER="$OPTS_ENTER -$OPT '$OPTARG'";;
+      f|F|T) OPTS_XIWI="$OPTS_XIWI -$OPT";;
+      \?) echo "$USAGE" 1>&2
+          exit 2;;
+    esac
+done
+shift "$((OPTIND-1))"
+
```
Comment (dnschneid) [inline host-bin/startxiwi]: Print usage and exit if there are no parameters at this point.

Comment (DennisLfromGA) [inline host-bin/startxiwi]: You mean like: if [ "$#" = "0" ]; then echo "$USAGE" 1>&2; exit 2 - after the 'shift...' ?

Comment (dnschneid) [inline host-bin/startxiwi]: Yep!

Comment (DennisLfromGA) [inline host-bin/startxiwi]: Okay, and thanx...

**Comment by dnschneid** [comment]
OK this works super-well.  Thanks!!

**Comment by dnschneid** [comment]
(And congrats on making it through the review gauntlet :) )

**Comment by DennisLfromGA** [comment]
I'm glad it's working but you deserve the credit, I can't believe your patience - and knowledge, just amazing!

I *love* xiwi and I think this'll really simplify things for those that use it.

----
@drinkcat has tried to teach me 'git' but I can't git ;-) used to it, I guess it'll take a lot more practice. I'm more comfortable using the github site, for now at least.

Thanx again,
-DennisL

----
P.S. I've been thinking about openning a new issue to request the use of 'pipeview' (`pv`) or something similar for things like archiving, restoring archives, and maybe deleting chroots. I think some more feedback when doing these ops would be pretty helpful - I do like 'spinner' though. ;)

Something I saw that might work is:

````
tar cf - $CHROOT -P |\
    pv -i 5 -berps $(du -sb $CHROOT |\
    awk '{print $1}') | gzip > $DATE-$CHROOT.tar.gz
````
Please let me know what you think.

Thanx.

**Comment by theblacklion** [comment]
+1 for pipeview. Using it for anything that takes longer than a glimpse..

**Pull request merged**